### ISSUE TITLE
Request to be(come) a commiter

### ISSUE THREAD

**Issue by rolandtritsch** [scoverage/scoverage.org#6]
Hi @ckipp01, et.al.,

Happy to fix this/get it up and running again.

Can you make me a committer on this repo.

Regards ...
    Roland

**Comment by ckipp01** [comment]
Unfortunately the last time I checked with @gslowikowski this domain is no longer owned by anyone in org. In my opinion if we were to have a public website again, it should be completely redone using something like mdoc, Laika, etc. This repo should probably be archived.

**Comment by rolandtritsch** [comment]
Was just looking for a place to document how all of the repos work together and to document how to contribute.

If we retire this repo, what is a/the central place to document stuff that is cross-repo? Maybe a README for `scoverage`?

**Comment by rolandtritsch** [comment]
And yes. Let's archive/retire this repo.

**Comment by rolandtritsch** [comment]
@sksamuel @ckipp01 Here is what I suggest to do ...

* Archive/Delete this repo
* Make the link at the top of the `scoverage` page point to `https://github.com/scoverage`
![image](https://user-images.githubusercontent.com/671381/189039893-b7fe8076-63f0-4e20-93b4-e379ca64bc7f.png)
* Write up a README for the `scoverage` page
  * Welcome to scoverage (what problem are we solving) 
  * Introduction to the components/repos
  * How to get started (pointing to existing documentation/material/tutorials/talks)
  * Example reports
  * How to contribute (general guidelines; can/need to be complemented with repo specific CONTRIBUTING.md files)
 
Happy to write up a draft for that README, Thoughts?

**Comment by sksamuel** [comment]
yep, I'll archive this repo now, and change the link, and can you make a PR for the readme
